pyproject.toml

```
@@ -9,7 +9,7 @@ exclude = "((.eggs | .git | .pytype | .pytest_cache | build | dist))"
 
 [tool.poetry]
 name = "questionary"
-version = "2.0.1"
+version = "2.1.0"
```

```suggestion
version = "2.0.1"
```

The version will be bumped when we make a new release.